content/doc/book/securing-jenkins.adoc

```diff
@@ -0,0 +1,50 @@
+---
+layout: simplepage
+---
+:doctitle: Securing Jenkins
+:notitle:
+:description:
+:email: jenkinsci-users@googlegroups.com
+:imagesdir: /doc/book/resources/securing-jenkins
+:toc: left
+
+== Securing Jenkins
+In the default configuration, Jenkins does not perform any security checks. This means the ability of Jenkins to
```

Comment:
This should be modified to note that this is true for Jenkins 1.x, but is not true for Jenkins 2.x